Resources/Computing/ARCComputingElement.py

```diff
@@ -334,9 +334,19 @@ def getCEStatus(self):
     else:
       # The system which works properly at present for ARC CEs that are configured correctly.
       # But for this we need the VO to be known - ask me (Raja) for the whole story if interested.
-      cmd = 'ldapsearch -x -LLL -H ldap://%s:2135 -b mds-vo-name=resource,o=grid "(GlueVOViewLocalID=%s)"' % (
-          self.ceHost, vo.lower())
-      res = shellCall(0, cmd)
+      # cmd = 'ldapsearch -x -LLL -H ldap://%s:2135 -b mds-vo-name=resource,o=grid "(GlueVOViewLocalID=%s)"' % (
+      #     self.ceHost, vo.lower())
+      cmd1 = 'ldapsearch -x -o ldif-wrap=no -LLL -h %s:2135  -b \'o=glue\' "(&(objectClass=GLUE2MappingPolicy)(GLUE2PolicyRule=vo:%s))"' % (self.ceHost, vo.lower())
+      if self.queue:
+        cmd2 = ' | grep GLUE2MappingPolicyShareForeignKey | grep %s' % (self.queue)
+      else:
+        gLogger.error('ARCComputingElement: No queue ...')
+      cmd3 = ' | sed \'s/GLUE2MappingPolicyShareForeignKey: /GLUE2ShareID=/\' | xargs -L1 ldapsearch -x -o ldif-wrap=no -LLL -h %s:2135 -b \'o=glue\'  | egrep \'(ShareWaiting|ShareRunning)\'' % (self.ceHost)
+      if self.cmd2 :
```

Comment:
https://github.com/DIRACGrid/DIRAC/pull/4658/checks?check_run_id=809227705#step:5:15

But as `cmd2` is defined inside `if`, it might not be defined at all and an exception would occur.
Maybe you just want
```python
if not self.queue:
  return S_ERROR(...
# build the command
```
?